pkg/trace/api/otlp.go

```diff
@@ -104,7 +106,21 @@ func NewOTLPReceiver(out chan<- *Payload, cfg *config.AgentConfig, statsd statsd
 func (o *OTLPReceiver) Start() {
 	cfg := o.conf.OTLPReceiver
 	if cfg.GRPCPort != 0 {
-		ln, err := net.Listen("tcp", fmt.Sprintf("%s:%d", cfg.BindHost, cfg.GRPCPort))
+		var ln net.Listener
+		var err error
+		if grpcFDStr, ok := os.LookupEnv("DD_OTLP_CONFIG_GRPC_FD"); ok {
```

Comment:
Any reason this is an env var only setting vs regular agent config ?

Reply:
Yes ! Those are set by the `loader` to pass to the `trace-agent`, they wouldn't make sense as config because you can't know their value ahead of times (so it's impossible to have them in the config file for example)
It's also not a value that the user should set, so it wouldn't really make sense as a config